DemoApp/ChatPresenter.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatUI
+import UIKit
+
+extension UIViewController {
+    // TODO: Where to put this???
+    func presentChat(apiKey: String, userCredentials: UserCredentials) {
```

Comment:
Oh right, it's not obvious because the other part is missing. We should be able to present the chat UI also from the screen where you can set your own credentials.